routers/web/org/members.go

```diff
@@ -108,13 +108,20 @@ func MembersAction(ctx *context.Context) {
 		}
 	case "leave":
 		err = models.RemoveOrgUser(org.ID, ctx.Doer.ID)
-		if organization.IsErrLastOrgOwner(err) {
+		if err == nil {
+			ctx.Flash.Success(ctx.Tr("form.organization_leave_success", org.DisplayName()))
+			ctx.JSON(http.StatusOK, map[string]interface{}{
+				"redirect": "",
```

Comment:
You need a valid `redirect` like here. At least, it could be the AppSubURL

Reply:
```suggestion
				"redirect": "", // keep the user stay on current page, in case they want to do other operations.
```

Hmm, it's done by purpose, so LGTM